sdk/ml/azure-ai-ml/azure/ai/ml/_internal/_utils/_yaml_utils.py

```diff
@@ -0,0 +1,20 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+
+import strictyaml
+
+
+class _SafeLoaderWithBaseLoader(strictyaml.ruamel.SafeLoader):
+    def fetch_comment(self, comment):
+        pass
+
+    def add_version_implicit_resolver(self, version, tag, regexp, first):
+        """Overwrite the method to use base resolver instead of version default resolver."""
+        self._version_implicit_resolver.setdefault(version, {})
```

Comment:
comment about version's meaning?

Reply:
May add some more comments on what does this mean by this line.

What's the diference with "base resolver" & "version default resolver"?

https://github.com/crdoconnor/strictyaml/blob/master/strictyaml/ruamel/resolver.py